src/lib.rs

```diff
@@ -165,4 +171,24 @@ impl CommonOptions {
         }
         Ok(config)
     }
+
+    fn opt_level(&self) -> wasmtime::OptLevel {
+        match (self.optimize, self.opt_level.clone()) {
+            (true, _) => wasmtime::OptLevel::Speed,
+            (false, other) => other,
+        }
+    }
+}
+
+fn parse_opt_level(opt_level: &str) -> Result<wasmtime::OptLevel> {
+    match opt_level {
+        "s" => Ok(wasmtime::OptLevel::SpeedAndSize),
+        "0" => Ok(wasmtime::OptLevel::None),
+        "1" => Ok(wasmtime::OptLevel::Speed),
```

Comment:
Is having 1 and 2 mean the same thing primarily for familiarity from other tool's optimization levels?

Reply:
Ah should have read the PR description before the diff :)

Reply:
I'm "ok" with this, but I don't necessarily think we need to unless we at some point think we'll have 1:1 matching levels with other tools.